V2_MIGRATION_GUIDE.md

```diff
@@ -6,6 +6,14 @@
 
 - Android API version 21 or later.
 
+## OIDC only
+
+Verson 2 only supports OIDC-compliant applications. When using this SDK to authenticate users, only the current Auth0 authentication pipeline and endpoints will be used.
```

Comment:
```suggestion
Version 2 only supports OIDC-Conformant applications. When using this SDK to authenticate users, only the current Auth0 authentication pipeline and endpoints will be used.
```

Reply:
Let's use the same wording as the dashboard -> Comformant

Reply:
👍 pushed a new commit to use consistent terminology.